ethereum/eip712/encoding.go

```diff
@@ -262,53 +251,22 @@ func validatePayloadMessages(msgs []sdk.Msg) error {
 		return errors.New("unable to build EIP-712 payload: transaction does contain any messages")
```

Comment:
Nit: update the description of this method, it says that checks that msgs are all same type